src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -131,6 +133,12 @@ class JesBackend extends Backend with LazyLogging {
     JesOutput(value, engineFunctions.gcsPathFromAnyString(value).toString, Paths.get(value))
   }
 
+  override def stdout(workflowId: WorkflowId, workflowName: String, callName: String): WdlFile =
+    WdlFile(s"$CromwellExecutionBucket/$workflowName/$workflowId/call-$callName/$LocalStdoutValue")
+
+  override def stderr(workflowId: WorkflowId, workflowName: String, callName: String): WdlFile =
+    WdlFile(s"$CromwellExecutionBucket/$workflowName/$workflowId/call-$callName/$LocalStderrValue")
```

Comment:
Or perhaps have both `stdout`/`stderr` call another function which takes either `LocalStdoutValue` or `LocalStderrValue` as well as the other stuff and does the right thing, something like that.